modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc

```diff
@@ -17,21 +16,21 @@
 
 . In the *Administrator* perspective of the web console, navigate to *Operators* -> *Installed Operators* to verify that the {gitops-title} Operator is installed.
 . Navigate to the {rh-app-icon} menu -> *OpenShift GitOps* -> *Cluster Argo CD*. The login page of the Argo CD UI is displayed in a new window.
-. Obtain the password for the Argo CD instance:
+. Optional: To log in with your {OCP} credentials, ensure you are a user of the `cluster-admins` group and then select the `LOG IN VIA OPENSHIFT` option in the Argo CD user interface.
++
+[NOTE]
+====
+To be a user of the `cluster-admins` group, use the `oc adm groups new cluster-admins <user>` command, where `<user>` is the default cluster role that you can bind to users and groups cluster-wide or locally. 
+====
```

Comment:
This looks like new content, did you take it from another module? Any actual new technical content, even as small as this one, needs SME/QE review.

Reply:
It is part of the bug that is already merged and closed. Just merging it from the main.